sky/backends/cloud_vm_ray_backend.py

```diff
@@ -48,6 +48,7 @@
 from sky.provision import instance_setup
 from sky.provision import metadata_utils
 from sky.provision import provisioner
+from sky.provision.kubernetes import utils as kubernetes_utils
```

Comment:
Can we quickly check if this global import does not break skypilot if kubernetes dependencies are not installed?

Reply:
Tested by running `sky launch --cloud gcp -y` from a new env where only `pip install -e ".[gcp]"` is set, and there was no issue raised.